make verify failed

<!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
**What happened**:
running cmd `make verify` failed, the error output is 
```
hack/verify-crds.sh
Start to install controller-gen tool
hack/../hack/generate-crds.sh: line 62: paths=./...: No such file or directory
make: *** [Makefile:82: verify-crds] Error 127 
```
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

--- a/hack/generate-crds.sh
+++ b/hack/generate-crds.sh
@@ -54,6 +54,7 @@ function :pre:install: {
   if [ "$(which controller-gen)" == "" ]; then
       echo "Start to install controller-gen tool"
       GO111MODULE=on go install -v sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2
+      export PATH=$PATH:$GOPATH/bin
   fi
 }
 

